### PR TITLE
[FEAT] : Activation Flow LedgerSync

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Activation Flow for Ledger Sync in LLM : 
- No trustchain and I want to create one
- Device Selection
- Follow Instruction on Device
- Success

https://github.com/user-attachments/assets/e8d87700-c6d0-4330-9fa1-4f40f471c965

### ❓ Context

- **JIRA or GitHub link**: 
  - [LIVE-12152]
  - [LIVE-12154]
  - [LIVE-12155]
  - [LIVE-12156]
  - [LIVE-12153]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-12152]: https://ledgerhq.atlassian.net/browse/LIVE-12152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-12154]: https://ledgerhq.atlassian.net/browse/LIVE-12154?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-12155]: https://ledgerhq.atlassian.net/browse/LIVE-12155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-12156]: https://ledgerhq.atlassian.net/browse/LIVE-12156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-12153]: https://ledgerhq.atlassian.net/browse/LIVE-12153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/locales/en/common.json [hunk 1]
```diff
@@ -6698,6 +6698,22 @@
...
+      "description": "You can retrieve your backup by simply connecting your Ledger to the Ledger Live mobile and desktop apps."
+    },
+    "deviceAction": {
+      "title": "Continue on your Ledger {{wording}}",
+      "description": "Follow the instruction which appear on your Ledger’s Trusted Display."
+    },
+    "success": {
+      "sync": "Synchronization successful!",
+      "syncDesc": "Changes in your accounts will now automatically appear across all apps and platforms.",
+      "activation": "Success!",
```
Comment (gre) [inline apps/ledger-live-mobile/src/locales/en/common.json:6713]: i find the `!`s a bit odd. is this really in the designs?

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+export const headerOptions: ReactNavigationHeaderOptions = {
+  headerShown: false,
+};
+
+type ChooseDeviceProps = Props & {
+  isFocused?: boolean;
+  goToFollowInstructions: (device: Device) => void;
+};
+
+const APP_NAME = "Ledger Sync";
```
Comment (gre) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx:37]: this should now be part of `hw-trustchain` lib

File: apps/ledger-live-mobile/src/locales/en/common.json (hunk 1 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/locales/en/common.json:6713]: https://www.figma.com/design/uI7PKnUK8nAZ0USyiEF3wQ/Wallet-Sync?node-id=6893-18566&t=N4KOimwx2hBW4G5p-0

yup

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx (hunk 2 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx:37]: Yep, I'll do it on a separate PR for both platforms

**Comment by mcayuelas-ledger** [comment]
https://github.com/LedgerHQ/ledger-live/pull/7338

**Review by KVNLS — APPROVED** [review]

**Pull request merged**